apps/probate/demo/sops-secrets/demo-postgres.enc.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+data:
+  values.yaml: ENC[AES256_GCM,data:7g0zBr6BCV38IN8BO7fy5fbEbJuhxpuxgcz3+Ewq3jcypVtgPIU8NsukpmaYb1wOCfWMpfAQ8MlqYJmQ90AY6qX4CG6xe7H8RwxcC0GYGJQ1M90eVHYT9eiWL30nCY0Gzyr4ryyrogExjeOvEFx4sshWGyhZXE6ao3OqZSsqKJtcj8y1,iv:j5VpOG1fVzqslggVqObddJk6EuVzcTjBxMJrCDaU/lg=,tag:vUmUQleHrFd8U3Glj3NIAA==,type:str]
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: prometheus-values
+  namespace: monitoring
+type: Opaque
+sops:
+  kms: []
+  gcp_kms: []
+  azure_kv:
+    - vault_url: https://dcdcftappssboxkv.vault.azure.net
```

Comment:
I think because its demo environment, you need to use SOPS key from this key vault  `dcdcftappsdemokv `